missing_numbers.py

```diff
@@ -0,0 +1,14 @@
+def missing_number(b):
+    if len(b) != 0:
```

Comment:
An empty list is falsy in python i.e 
```python
# instead of writing 
if len(some_list) != 0:
  # do you favourite thing here

# you can write 
if some_list:
   # do something kind here
```
> If a list is empty, its resolves to false